genkit-tools/cli/src/utils/manager-utils.ts

```diff
@@ -85,3 +96,61 @@ export async function runWithManager(
     logger.error(`${error.stack}`);
   }
 }
+
+/**
+ * Fetches an existing Telemetry Server URL from servers config. Otherwise,
+ * creates a new server and updates the server config.
+ *
+ * @returns telemetry server url
+ */
+async function getOrStartTelemetryServer(): Promise<string> {
```

Comment:
I think this needs to be conditional,no?

Reply:
I mean, what if I do `genkit start --` twice in the same folder? the second one won't start the telemetry server, but if you shut down the first the runtime will be without telemetry server

Reply:
The approach we discussed: when doing `genkit start --` set GENKIT_TELEMETRY_SERVER_URL env var runtime ignores notify calls... don't like that?

Reply:
Not that I don't like it, @apascal07 suggested using `resolveTelemetryServer` to check if there is a running server. 

Also, doesn't the env var approach also face the same problem as your second comment? Killing the first `genkit start` will kill the telemetry server but the second one is left without one.

Reply:
No. With env var the runtime would be permanently bound to the `genkit start` cli instance that spawned it. It will always have the telemetry server.